addons/aws/external-dns/variables.tf

```diff
@@ -0,0 +1,74 @@
+variable "cloud" {
+  default = null
+}
+
+variable "cluster_id" {
+  default = null
+}
+
+variable "name_prefix" {
+  description = "name prefix"
+}
+
+variable "output_path" {
+  default = "../../../outputs"
+}
+
+variable "tsb_image_sync_username" {
+}
+
+variable "aws_k8s_regions" {
+  default = []
+}
+
+locals {
+  k8s_regions = var.aws_k8s_regions
+}
+
+variable "tetrate_owner" {
+  default = null
+}
+variable "tetrate_team" {
+  default = "tetrate-service-bridge-sandbox"
+}
```

Comment:
These two should be mandatory and not provided by default. We need to enforce proper ownership tagging. Everyone should configure their team and owner name.